spel/scripts/amigen8-build.sh

```diff
@@ -69,8 +68,78 @@ function err_exit {
    fi
 }
 
+
+# CentOS 8 repos
+DEFAULTREPOS=(
+    baseos
+    extras
+    appstream
+)
+if rpm --quiet -q redhat-release
```

Comment:
Interesting.  This syntax actually works?

Reply:
yep, if all you care about is the exit status, and 0 means success and anything else means failure

Reply:
on redhat, it echos:
```
# if rpm --quiet -q redhat-release; then echo it is redhat; fi
it is redhat
# <new prompt>
```

on centos, no echo:
```
# if rpm --quiet -q redhat-release; then echo it is redhat; fi
# <new prompt>
```

Reply:
😎